lib/myinfo/v3/person_basic.rb

```diff
@@ -17,15 +17,15 @@ def initialize(nric_fin:, txn_no: nil, attributes: nil)
       def call
         super do
           headers = header(params: params)
-          endpoint_url = "/#{slug}?#{params.to_query}"
+          endpoint_url = "/#{slug(gateway: true)}?#{params.to_query}"
 
           response = http.request_get(endpoint_url, headers)
           parse_response(response)
         end
       end
 
-      def slug
-        "gov/v3/person-basic/#{nric_fin}/"
+      def slug(gateway: false)
```

Comment:
> i'm thinking we can actually split the slug the one being used for signing and one for initiating the api call, instead of deciding whether to pass the gateway flag.
> 
> it will be easier to understand than doing code tracing in the code to determine what was used.
> 
> what do you think?

Ah, I thought to handle in a single method by passing an attribute so that, we don't need to have 2 methods for the same cause.